src/preferences/PreferencesManager.js

```diff
@@ -35,6 +35,7 @@ define(function (require, exports, module) {
     var OldPreferenceStorage    = require("preferences/PreferenceStorage").PreferenceStorage,
         AppInit                 = require("utils/AppInit"),
         Commands                = require("command/Commands"),
+        ProjectManager          = require("project/ProjectManager"),
```

Comment:
I'm not sure you want to get an import loop going here... I guess it's okay because nothing is using `ProjectManager` at the root of the file, but these kinds of cross-module dependencies tend to increase over time and that has led to tricky to figure out bugs.